content/blog/2025/community-ownership/index.md

```diff
@@ -0,0 +1,74 @@
+
+Being a platform and service provider gives us a lot of power, and also introduces a potential source of _lock-in_ for our member communities. While 2i2c's organizational mission and culture is strongly aligned with open infrastructure, we believe it's important to encode commitments like these in a formal way to provide both transparency and accountability to our member communities.
+
+## Our commitment to open technology
+
+Below we copy the original language of this policy from our [Commitment to Open Technology](../../../open-technology/index.md):
+
+<!-- TODO: When we switch to MyST, we should embed this rather than copy/paste -->
+
+_Definitions of MUST, MUST NOT, SHOULD, MAY, etc are defined in [RFC 2119](https://tools.ietf.org/html/rfc2119)_
+
+1. All engineering artifacts (code, documentation, etc) produced by 2i2c's engineering team MUST be licensed under an open source license approved by a non-profit organization that is not 2i2c.
+2. Open Source Projects originating at 2i2c, or stewarded by 2i2c, MUST NOT require a [Contributor Licensing Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement) that includes Copyright Assignment to 2i2c. 
+3. The list of external organizations that define licenses we accept are
+    1. [the Open Source Initiative](https://opensource.org/)
+    2. the [Organization for Ethical Source](https://ethicalsource.dev/). 
+4. Modifying (1), (2), or (3) MUST be done through a 2/3 majority vote of 2i2c staff. 
+
+## What does this commitment mean?
+
+In plain language, here's what this commitment means:
+
+1. We'll only use open source licenses that have been approved by standard non-profits that are broadly recognized by the tech industry.
+2. For anything we build, we won't require contributors to give up the rights to their contributions via CLAs, so that it is much harder for 2i2c to change our licenses in the future.
+3. Changing this policy will require organization-wide agreement, and in the future we'll give authority over this policy to a group of people representing our member communities.
+
+## Why are licenses and CLAs important? 
+
+Many organizations claim to be committed to open infrastructure, while retaining the ability to _change this commitment in the future when it is in their interests_. A classic example of this is a "bait and switch" that looks something like this:
+
+1. A company releases software under an open source license and professes to build an open source community around it.
+2. However, they retain the rights to all of the code in their projects through a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement) (CLA). This generally means that contributors must _give up the rights to their contribution_ in order to make that contribution.
```

Comment:
```suggestion
2. However, they retain the rights to all of the code in their projects through a [Contributor License Agreement](https://en.wikipedia.org/wiki/Contributor_License_Agreement) (CLA) with copyright assignment. This generally means that contributors must _give up the rights to their contribution_ in order to make that contribution.
```

Reply:
Since you can have CLAs without copyright assignment technically and often this turns into a whataboutism